### PR TITLE
experimental: cfw deployment guide restructure

### LINKED ISSUE #4218: Update 'Deploy to Cloudflare Workers' guide
Update https://www.prisma.io/docs/guides/deployment/deployment-guides/deploying-to-cloudflare-workers:
- Bring language up to style guide standard

@ObamaTheLlama114 tested the steps as part of #3188 , so we know that they work.

### PR DESCRIPTION
**Pull request by ruheni**
Cloudflare Workers deployment guide example

Closes #4218
Closes https://github.com/prisma/team-orm/issues/145

### CODE REVIEW

**Comment by Jolg42** [comment]
Note: I think we should mention explicitly what our guide uses, which is either
- the "old" Service Worker format
- the "new" ES modules format 
https://developers.cloudflare.com/workers/learning/migrate-to-module-workers/

**Comment by ruheni** [comment]
We're rethinking the structure of our deployment guides and intend for them to only focus on the deployment rather than development/ integration. The integration with Cloudflare Workers guide will likely be separate.

File: content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx [hunk 1]
```diff
@@ -1,329 +1,75 @@
...
-npx prisma generate --data-proxy
+ + DIRECT_DATABASE_URL = "prisma://accelerate.prisma-data.net/?api_key=•••••••••••••••••"
 ```
 
-The generated Client has a smaller bundle size and is optimized for edge environments like Cloudflare Workers.
+1. Generate Prisma Client that connects through Accelerate over HTTP:
 
-The smaller bundle size is due to the fact that the interfaces talking to the database (the [Prisma engines](/concepts/components/prisma-engines)) are no longer bundled with Prisma Client as this logic is now handled by the Data Proxy.
+   ```terminal
+   npx prisma generate --data-proxy
```
Comment (ankur-arch) [inline content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx:53]: For versions above or equal to 5.0.0, you can use the following flag to generate the client  for Accelerate

```suggestion
   npx prisma generate --accelerate
```

File: content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx [hunk 2]
```diff
@@ -1,329 +1,75 @@
...
-
-This data model will be used to store incoming requests from your Worker.
-
-## 5. Create a repository and push to GitHub
-
-To prepare for the steps ahead, let's [create a private repository](https://github.com/new) on GitHub.
+### Update your project
 
-![Create a private repository](../../../doc-images/guides/deploy/create-repo.png)
+1. Create a new environment variable `DIRECT_DATABASE_URL` and set the Accelerate connection string as it's value
```
Comment (janpio) [inline content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx:36]: Why Accelerate URL as `DIRECT`!?

**Comment by ruheni** [comment]
Updated via #5377

**Pull request closed without merging**